server/src/main/scala/controllers/Application.scala

```diff
@@ -325,20 +327,54 @@ class Application @Inject()(implicit val config: Configuration,
       def getKeyCloakUsers(): Future[List[KeyCloakUser]] = {
         log.info(s"Got these roles: ${getLoggedInUser().roles}")
         if (getLoggedInUser().roles.contains(ManageUsers)) {
-          keyCloakClient.getUsersNotInGroup("Approved")
+          keyCloakClient.getUsers()
+        } else throw new Exception("You do not have permission manage users")
```

Comment:
Extract this message string to prevent them diverging by accident in future as it's used in three places